_posts/2024-06-12-release.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Release 2024.24
+author: Priya
+date: 2024-06-12
+category: Jekyll
+layout: post
+---
+### Enhancement:
+
+#### Combine Dataset Feature Improvements:
+Mammoth now provides full visibility of column names without the need for hovering, saving time and improving efficiency. Scrolling has been reduced to easily locate unmatched columns, and the search functionality has been added for quicker identification of specific columns.
```

Comment:
Combine dataset modal shows longer column names as wrapped in multiple lines. This takes away the need to hover ellipsised names. Search for column names has been added. Also, a selector for unmatched columns is put for easier access.